Java/bit_manipulation/swap.java

```diff
@@ -0,0 +1,26 @@
+//Program to swap two numbers using bit manipulation.
+import java.util.Scanner;
+public class swap {
+
+	public static void main(String[] args) {
+		Scanner sc=new Scanner(System.in);
+		int a=sc.nextInt();
+		int b=sc.nextInt();
+		//swapping 
+		a=a^b;
+		b=a^b;
+		a=a^b;
+		System.out.println(a);
+		System.out.println(b);
+
+	}
+}
+
+
+Example No1:- Let a=5 and b=10 

```

Comment:
Please go through some recently merged Java PRs to get a better idea of the standard Java Template that we use.